File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Cpls/CPL0106U00GrdListItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Cpls
{
    [Serializable]
    public class CPL0106U00GrdListItemInfo
    {
        private String _groupGubunA;
        private String _hangmogCode;
        private String _specimenCode;
        private String _emergency;
        private String _subHangmogCode;
        private String _subSpecimenCode;
        private String _specimenName;
        private String _subEmergency;
        private String _gumsaName;
        private String _continueYn;
        private String _groupGubunB;
        private String _sgCode;
        private String _rowState;

        public String GroupGubunA
        {
            get { return this._groupGubunA; }
            set { this._groupGubunA = value; }
        }

        public String HangmogCode
        {
            get { return this._hangmogCode; }
            set { this._hangmogCode = value; }
        }

        public String SpecimenCode
        {
            get { return this._specimenCode; }
            set { this._specimenCode = value; }
        }

        public String Emergency
        {
            get { return this._emergency; }
            set { this._emergency = value; }
        }

        public String SubHangmogCode
        {
            get { return this._subHangmogCode; }
            set { this._subHangmogCode = value; }
        }

        public String SubSpecimenCode
        {
            get { return this._subSpecimenCode; }
            set { this._subSpecimenCode = value; }
        }

        public String SpecimenName
        {
            get { return this._specimenName; }
            set { this._specimenName = value; }
        }

        public String SubEmergency
        {
            get { return this._subEmergency; }
            set { this._subEmergency = value; }
        }

        public String GumsaName
        {
            get { return this._gumsaName; }
            set { this._gumsaName = value; }
        }

        public String ContinueYn
        {
            get { return this._continueYn; }
            set { this._continueYn = value; }
        }

        public String GroupGubunB
        {
            get { return this._groupGubunB; }
            set { this._groupGubunB = value; }
        }

        public String SgCode
        {
            get { return this._sgCode; }
            set { this._sgCode = value; }
        }

        public String RowState
        {
            get { return this._rowState; }
            set { this._rowState = value; }
        }

        public CPL0106U00GrdListItemInfo() { }

        public CPL0106U00GrdListItemInfo(String groupGubunA, String hangmogCode, String specimenCode, String emergency, String subHangmogCode, String subSpecimenCode, String specimenName, String subEmergency, String gumsaName, String continueYn, String groupGubunB, String sgCode, String rowState)
        {
            this._groupGubunA = groupGubunA;
            this._hangmogCode = hangmogCode;
            this._specimenCode = specimenCode;
            this._emergency = emergency;
            this._subHangmogCode = subHangmogCode;
            this._subSpecimenCode = subSpecimenCode;
            this._specimenName = specimenName;
            this._subEmergency = subEmergency;
            this._gumsaName = gumsaName;
            this._continueYn = continueYn;
            this._groupGubunB = groupGubunB;
            this._sgCode = sgCode;
            this._rowState = rowState;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/cpls/CPL0106U00GrdListItemInfo.java
package nta.med.data.model.ihis.cpls;

public class CPL0106U00GrdListItemInfo {
	private String groupGubunA;
	private String hangmogCode;
	private String specimenCode;
	private String emergency;
	private String subHangmogCode;
	private String subSpecimenCode;
	private String specimenName;
	private String subEmergency;
	private String gumsaName;
	private String continueYn;
	private String groupGubunB;
	private String sgCode;
	
	public CPL0106U00GrdListItemInfo(String groupGubunA, String hangmogCode,
			String specimenCode, String emergency, String subHangmogCode,
			String subSpecimenCode, String specimenName, String subEmergency,
			String gumsaName, String continueYn, String groupGubunB,
			String sgCode) {
		super();
		this.groupGubunA = groupGubunA;
		this.hangmogCode = hangmogCode;
		this.specimenCode = specimenCode;
		this.emergency = emergency;
		this.subHangmogCode = subHangmogCode;
		this.subSpecimenCode = subSpecimenCode;
		this.specimenName = specimenName;
		this.subEmergency = subEmergency;
		this.gumsaName = gumsaName;
		this.continueYn = continueYn;
		this.groupGubunB = groupGubunB;
		this.sgCode = sgCode;
	}
	public String getGroupGubunA() {
		return groupGubunA;
	}
	public void setGroupGubunA(String groupGubunA) {
		this.groupGubunA = groupGubunA;
	}
	public String getHangmogCode() {
		return hangmogCode;
	}
	public void setHangmogCode(String hangmogCode) {
		this.hangmogCode = hangmogCode;
	}
	public String getSpecimenCode() {
		return specimenCode;
	}
	public void setSpecimenCode(String specimenCode) {
		this.specimenCode = specimenCode;
	}
	public String getEmergency() {
		return emergency;
	}
	public void setEmergency(String emergency) {
		this.emergency = emergency;
	}
	public String getSubHangmogCode() {
		return subHangmogCode;
	}
	public void setSubHangmogCode(String subHangmogCode) {
		this.subHangmogCode = subHangmogCode;
	}
	public String getSubSpecimenCode() {
		return subSpecimenCode;
	}
	public void setSubSpecimenCode(String subSpecimenCode) {
		this.subSpecimenCode = subSpecimenCode;
	}
	public String getSpecimenName() {
		return specimenName;
	}
	public void setSpecimenName(String specimenName) {
		this.specimenName = specimenName;
	}
	public String getSubEmergency() {
		return subEmergency;
	}
	public void setSubEmergency(String subEmergency) {
		this.subEmergency = subEmergency;
	}
	public String getGumsaName() {
		return gumsaName;
	}
	public void setGumsaName(String gumsaName) {
		this.gumsaName = gumsaName;
	}
	public String getContinueYn() {
		return continueYn;
	}
	public void setContinueYn(String continueYn) {
		this.continueYn = continueYn;
	}
	public String getGroupGubunB() {
		return groupGubunB;
	}
	public void setGroupGubunB(String groupGubunB) {
		this.groupGubunB = groupGubunB;
	}
	public String getSgCode() {
		return sgCode;
	}
	public void setSgCode(String sgCode) {
		this.sgCode = sgCode;
	}
}
